cli/seed_proto.json

```diff
@@ -0,0 +1,59 @@
+{
+    "languageLanguageRef": "/Users/josh/dev/language-persistence/build/bundle.js",
+    "perspectiveDiffSync": {
```

Comment:
`perspectiveDiffSync` is different to the other languages in that it is the name of a specific link language implementation, while the other languages need to have exactly one - and `agentLanguage` for instance describes the role of that language.

We could have this be an array with the name "linkLanguages" and then we could really fill the `knownLinkLanguages` array with multiple.

Reply:
yeah right; ill rename the key to linkLanguages and make it an array